### DISCUSSION TITLE
LICENSE DOUBTS

### DISCUSSION THREAD

**Discussion by ZSY4** [immich-app/immich#11328] [Q&A] [answered]
Will not purchasing a license affect the normal use of immich?
LICENSEThis makes me wonder, is the LICENSE simply donated to support or does not buy it affect daily use?

**Comment by zackpollard** [comment, marked as answer]
Hey, we have already made changes to the wording as you can see here https://github.com/immich-app/immich/discussions/11313

You are free to use the software without purchasing, with no paywalls, ever. However, we do ask that if you find value in the software and are able to, you purchase immich. Have a great rest of your day!
